learn/update_and_migration/updating.mdx

```diff
@@ -36,13 +36,124 @@ Once the project has been successfully updated, you will receive an email confir
 
 ## Updating a self-hosted Meilisearch instance
 
+You may update a self-hosted instance in one of two ways: with or without a dump.
+
 <Capsule intent="warning">
 This guide only works for v0.15 and above. If you are using an older Meilisearch release, please [contact support](https://discord.meilisearch.com) for more information.
 </Capsule>
 
-### Step 1: Export data
+### Dumpless upgrade <NoticeTag type="experimental" label="experimental" />
+
+Dumpless upgrades are available when upgrading from Meilisearch >=v1.12 to Meilisearch >=v1.13
+
+#### Step 1: Make a backup
+
+Dumpless upgrades are an experimental feature. Because of that, it may in rare occasions partially fail and result in a corrupted database. To prevent data loss, create a snapshot of your instance:
+
+```sh
+curl \
+  -X POST 'MEILISEARCH_URL/snapshots'
+```
+
+Once the snapshot is available in your [snapshots folder](/learn/self_hosted/configure_meilisearch_at_launch#snapshot-destination), proceed to the next step.
```

Comment:
Shouldn't we tell people to look at their tasks instead of looking at the directory?